pristine-zen-garden.html

```diff
@@ -0,0 +1,274 @@
+                    <a href="/examples/index" title="This page's source HTML code, not to be modified.">HTML</a> and
+                    <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">CSS</a> to work on a copy locally. Once you have completed your
+                    masterpiece (and please, don&#8217;t submit half-finished work)
+                    upload your
+                    <abbr title="Cascading Style Sheets">CSS</abbr> file to a web server under your control.
+                    <a href="http://www.mezzoblue.com/zengarden/submit/"
+                        title="Use the contact form to send us your CSS file">Send us a link</a> to an archive of that file and all associated
+                    assets, and if we choose to use it we will download it and place
+                    it on our server.</p>
+            </div>
+
+            <div class="benefits" id="zen-benefits" role="article">
+                <h3>Benefits</h3>
+                <p>Why participate? For recognition, inspiration, and a resource we
+                    can all refer to showing people how amazing
+                    <abbr title="Cascading Style Sheets">CSS</abbr> really can be. This site serves as equal parts inspiration
+                    for those working on the web today, learning tool for those who
+                    will be tomorrow, and gallery of future techniques we can all
+                    look forward to.</p>
+            </div>
+
+            <div class="requirements" id="zen-requirements" role="article">
+                <h3>Requirements</h3>
+                <p>Where possible, we would like to see mostly
+                    <abbr title="Cascading Style Sheets, levels 1 and 2">CSS 1 &amp; 2</abbr> usage.
+                    <abbr title="Cascading Style Sheets, levels 3 and 4">CSS 3 &amp; 4</abbr> should be limited to widely-supported elements
+                    only, or strong fallbacks should be provided. The CSS Zen Garden
+                    is about functional, practical
+                    <abbr title="Cascading Style Sheets">CSS</abbr> and not the latest bleeding-edge tricks viewable by
+                    2% of the browsing public. The only real requirement we have
+                    is that your
+                    <abbr title="Cascading Style Sheets">CSS</abbr> validates.</p>
+                <p>Luckily, designing this way shows how well various browsers have
+                    implemented
+                    <abbr title="Cascading Style Sheets">CSS</abbr> by now. When sticking to the guidelines you should
+                    see fairly consistent results across most modern browsers. Due
+                    to the sheer number of user agents on the web these days &#8212;
+                    especially when you factor in mobile &#8212; pixel-perfect layouts
+                    may not be possible across every platform. That&#8217;s okay,
```

Comment:
Should there be That&#8217;s or does it need to be edited?

Reply:
&*8217; is the escape character in html. It will show up as ' in the browser